scout/adapter/mongo/variant_evaluation_term.py

```diff
@@ -0,0 +1,175 @@
+"""Query and updates tags and dismiss nomenclature."""
+
+import logging
+from datetime import datetime
+
+import pymongo
+
+from scout.constants.case_tags import TRACKS
+from scout.constants.variant_tags import EVALUATION_TERM_CATEGORIES
+
+LOG = logging.getLogger(__name__)
+
+
+class EvaluationTerm(object):
+    """Represents a variant evaluation term"""
+
+    def __init__(self, term_obj):
+        self.key = term_obj.get("term_key")  # mandatory
```

Comment:
Using characters as keys does not work properly.

A term like this loads but the use cant select it as a evaluation term

` {"key": 1, "label": "B", "name": "Benign", "description": "Benign, does not cause phenotype", "label_class": "success"},`

Scout only reports it as a term with key -1

![image](https://user-images.githubusercontent.com/12696913/124099377-bc3f2280-da5d-11eb-8367-f10232ffcc00.png)

Reply:
Yeah I know, I think also master behaves this way, I'll see if it can print a more informative message, or just "Updated variant tag"!

Reply:
This should be fixed now

Reply:
👍

Reply:
Still work in progress tho, I'll let you know when I'm done with the changes